mreg/log_processors.py

```diff
@@ -44,7 +45,10 @@ def filter_sensitive_data(_: Any, __: Any, event_dict: EventDict) -> EventDict:
         event: str = event_dict["event"]
 
         if event == "request" and "password" in content:
-            event_dict["content"]["password"] = '...'
+            if isinstance(event_dict["content"],dict):
+                event_dict["content"]["password"] = '...'
+            if isinstance(event_dict["content"],str):
```

Comment:
Should this be `elif`? It won't matter for the result, but it looks a bit weird to me that we run the test twice when only one result can happen (unless we in the previous test assign event_dict["content"] to a str.

Reply:
Yes, it should. Fixed in https://github.com/unioslo/mreg/commit/a7c378b34c56933886db1bf946e6be50a73dc38e